Developer-Guide.md

```diff
@@ -75,6 +78,22 @@ You need to install the following to build Kata Containers components:
 - `make`.
 - `gcc` (required for building the shim and runtime).
 
+# Individual component compatibility
+
+Using newer versions of individual components than the last offical released
+version may result in
+[compatibility issues](Releases.md#component-version-compatibility).
+
+If you have only updated a subset of the Kata components and notice strange
+behaviour, we recommend you update the remaining components to ensure you are
+using the latest versions of all of them. Remember to also [create a new image
+containing the latest agent
+version](#create-and-install-rootfs-and-initrd-image).
+
+If this still does not work, refer to the
```

Comment:
Suggested rewrite: If you still have compatibility issues after following the previous instructions, refer to the